app/models/my_module.rb

```diff
@@ -355,9 +359,78 @@ def get_new_position
     { x: 0, y: positions.last[1] + HEIGHT }
   end
 
+  def toggle_tab(tab)
+    toggled = true
+
+    begin
+      # This is done in advance so even if enabling tabs
+      # (where can_uncheck doesn't need to be called),
+      # it's called anyway just to check if the tab
+      # parameter is legal (e.g. it's not an evil,
+      # injected value)
+      can_uncheck = send("can_uncheck_tab_#{tab}?")
```

Comment:
@Ducz0r is this intentionally left as is, or can I cahnge it with 'Object#send'?